dev-libs/libgit2/libgit2-0.24.1.recipe

```diff
@@ -0,0 +1,92 @@
+SUMMARY="A portable, pure C implementation of the Git core methods"
+DESCRIPTION="The library provides:
+* SHA conversions, formatting and shortening
+* abstracted ODB backend system
+* commit, tag, tree and blob parsing, editing, and write-back
+* tree traversal
+* revision walking
+* index file (staging area) manipulation
+* reference management (including packed references)
+* config file management
+* high level repository management
+* thread safety and reentrancy
+* descriptive and detailed error messages
+* ...and more (over 175 different API calls)"
+HOMEPAGE="http://libgit2.github.com/"
+COPYRIGHT="2005-2016 libgit2 project"
+LICENSE="GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/libgit2/libgit2/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="60198cbb34066b9b5c1613d15c0479f6cd25f4aef42f7ec515cd1cc13a77fede"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	libgit2$secondaryArchSuffix = $portVersion
+	lib:libgit2$secondaryArchSuffix = $portVersion compat >= 0
```

Comment:
It seems the version of the lib is not **`0.24.1`** but **`0.24.0`**. So this line should probably be:
```
	devel:libgit2$secondaryArchSuffix = 0.24.0 compat >= 0
```